gap/OscarInterface/PackageInfo.g

```diff
@@ -67,7 +67,7 @@ AbstractHTML   :=  "",
 PackageDoc := [],
 
 Dependencies := rec(
-  GAP := ">= 4.12",
+  GAP := ">= 4.14",
```

Comment:
Do we need to keep this in sync with the used gap version? If yes, we forgot that when we upgraded to 4.13. If no, we can just revert the change.
```suggestion
  GAP := ">= 4.12",
```

Reply:
Logically, this dependency describes which minimal GAP version is needed by the code in the GAP package OscarInterface. From that point of view, "4.12" is probably still correct.
On the other hand, the dependencies of Oscar.jl prescribe the GAP.jl version, the dependencies of GAP.jl determine the GAP version, and if some code in OscarInterface does not work because it needs a newer GAP version then we will notice this already when this code gets added.
Thus I would propose to remove the GAP dependency here, for the sake of simplicity.

Reply:
```suggestion
```
so you propose to apply this patch, right?

Let's hear @fingolfin opinion as well

Reply:
I am fine with changing this